src/main/resources/org/eolang/speco/1-2-specialization.xsl

```diff
@@ -89,6 +89,7 @@ SOFTWARE.
       <xsl:for-each select="/program/objects/o[@name=$name]">
         <xsl:call-template name="format">
           <xsl:with-param name="name" select="$objname"/>
+          <xsl:with-param name="spec" select="$name"/>
```

Comment:
@advasileva why do you need a separate param? Can you use `name` instead?

Reply:
@volodya-lombrozo Firstly, this change is already in the wizard, I do not know why the diff has not been updated. Secondly, `objname` is put in the `name` attribute (for example, `"foo_spec_x_first"`), and `name` is put in the `spec` attribute, which is the name of the specialized object (for example, `"foo"`). This is done to recursively create names like `"foo_spec_x_first_spec_y_first_spec_z_first"`. I can do this with one parameter, but the question is about the readability and maintainability of the code.

Reply:
@advasileva Maybe we need to merge changes from master branch to [advasileva:fix/tests-rule-3](https://github.com/advasileva/speco/tree/fix/tests-rule-3), or you can rebase?

Reply:
@volodya-lombrozo No, I can't `merge` or `rebase` because this branch has already been rebased from a branch identical to the current `master`. I tried to make a fake commit, but git didn't recalculate the diff with master